### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by tmuehl**
@v-stepanov @valgog @jkliff

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/36" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Install python requirements
 
 Run ZooKeeper and Kafka docker images
 
-    make --directory local-karka run
+    make -C local-test run
```
Comment (v-stepanov) [inline README.md]: why -C?
As for me --directory is more informative.

Comment (jkliff) [inline README.md]: are we writing a make tutorial?

Comment (valgog) [inline README.md]: 😂

Comment (tmuehl) [inline README.md]: Ok, ok, I change it back to --directoy...
@jkliff That's what Nakadi is about, isn't it? :)

**Pull request merged**